docs/sources/shared/datasources/tempo-search-traceql.md

```diff
@@ -105,8 +105,8 @@ To add a tag, follow these steps:
 ### Optional: Use Aggregate by
 
 {{% admonition type="warning" %}}
-**Aggregate by** is an [experimental feature](/docs/release-life-cycle/). Engineering and on-call support is not available. Documentation is either limited or not provided outside of code comments. No SLA is provided.
-[Enable the `metricsSummary` feature toggle](/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/) in Grafana to use this feature. Your Grafana Tempo data source must also point to a Tempo database with the [Metrics Summary API](https://grafana.com/docs/tempo/latest/api_docs/metrics-summary/) enabled. Contact Grafana Support to enable this feature in Grafana Cloud.
+Metrics summary API and the **Aggregate by** feature are deprecated in Grafana Cloud and Grafana 11.3 and later.
+It will be removed in a future release.
```

Comment:
This type of admonition goes against [timeless documentation](https://developers.google.com/style/timeless-documentation), but I am guessing it makes sense to have it here.